File: fix_quotes.js
const fs = require('fs');
const path = require('path');

function walkSync(dir, filelist = []) {
    const files = fs.readdirSync(dir);
    for (const file of files) {
        const filepath = path.join(dir, file);
        if (fs.statSync(filepath).isDirectory()) {
            walkSync(filepath, filelist);
        } else {
            filelist.push(filepath);
        }
    }
    return filelist;
}

const files = walkSync('app/admin').filter(f => f.endsWith('.tsx') || f.endsWith('.ts'));
for (const file of files) {
    const content = fs.readFileSync(file, 'utf8');
    if (content.includes('\\"')) {
        console.log(`Fixing ${file}`);
        fs.writeFileSync(file, content.replace(/\\"/g, '"'), 'utf8');
    }
}
console.log('Done!');

File: fix_quotes.py
import os

for root, dirs, files in os.walk('app/admin'):
    for f in files:
        if f.endswith('.tsx') or f.endswith('.ts'):
            path = os.path.join(root, f)
            with open(path, 'r', encoding='utf-8') as file:
                content = file.read()
            if '\\"' in content:
                print(f"Fixing {path}")
                content = content.replace('\\"', '"')
                with open(path, 'w', encoding='utf-8') as file:
                    file.write(content)
print("Done!")
